### PR TITLE
000 - Authentication and authorisation required to access frontend and backend

### PR DESCRIPTION
**Pull request by AZZohaib**

### CODE REVIEW

File: src/controllers/AuthController.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import { getToken } from "../services/AuthService";
+
+export const getLoginForm = async (req: express.Request, res: express.Response): Promise<void> => {
+    res.render('loginForm.html');
+}
```
Comment (JennicaMistry) [inline src/controllers/AuthController.ts:6]: Needs validation

**Review by JennicaMistry — CHANGES_REQUESTED** [review]
Just check, have you updated accessibility json for the new page.

File: src/controllers/AuthController.ts (hunk 1 above)
Comment (AZZohaib) [reply src/controllers/AuthController.ts:6]: Added

**Review by JennicaMistry — APPROVED** [review]

**Review by JennicaMistry — APPROVED** [review]
Put it in a separate AuthValidator next time.

**Pull request merged**